### PR TITLE
Remove NUM_STREAMS and performance hint setting in llm_bench

### PR DESCRIPTION
**Pull request by peterchen-intel**
NUM_STREAMS should be controlled by CPU/GPU plugin, so that CPU/GPU plugin can decide what is the best in different secnarios

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
Should we merge it?

**Comment by peterchen-intel** [comment]
> Should we merge it?
We can merge it now. :)

File: llm_bench/python/utils/model_utils.py [hunk 1]
```diff
@@ -96,9 +96,6 @@ def set_default_param_for_ov_config(ov_config):
     # With this PR https://github.com/huggingface/optimum-intel/pull/362, we are able to disable model cache
     if 'CACHE_DIR' not in ov_config:
         ov_config['CACHE_DIR'] = ''
-    # OpenVINO self have default value 2 for nstreams on machine with 2 nodes. Reducing memory consumed via changing nstreams to 1.
```
Comment (ilya-lavrenov) [inline llm_bench/python/utils/model_utils.py:99]: can we also drop `PERFORMANCE_HINT` as `LATENCY` is a default for plugin?

Comment (peterchen-intel) [reply llm_bench/python/utils/model_utils.py:99]: Do we have a conformance test to verify this LATENCY as default performance hint for plugin?

Comment (sbalandi) [reply llm_bench/python/utils/model_utils.py:99]: No, we don't check that `LATENCY` is default for `PERFORMANCE_HINT ` now , but we have tests for def values ​​and i will add this property there

**Review by ilya-lavrenov — APPROVED** [review]

File: llm_bench/python/utils/model_utils.py (hunk 1 above)
Comment (sbalandi) [reply llm_bench/python/utils/model_utils.py:99]: [PR#25149](https://github.com/openvinotoolkit/openvino/pull/25149)

**Pull request merged**